bin/sozo/src/commands/dev.rs

```diff
@@ -244,9 +248,23 @@ where
         return Err(anyhow!("Build project using `sozo build` first"));
     }
 
-    let new_manifest =
+    let mut new_manifest =
         BaseManifest::load_from_path(&manifest_dir.join(MANIFESTS_DIR).join(BASE_DIR))?;
 
+    if let Some(skip_manifests) = skip_migration {
+        for contract_or_model in skip_manifests {
+            if let Some(index) =
+                new_manifest.contracts.iter().position(|c| c.name == contract_or_model)
```

Comment:
We may use [retain](https://doc.rust-lang.org/std/vec/struct.Vec.html#method.retain).